pkg/operators/v1alpha1/catalogsource_types.go

```diff
@@ -107,7 +107,7 @@ type GrpcPodConfig struct {
 	// If not specified, the pod priority will be default or zero if there is no
 	// default.
 	// +optional
-	PriorityClassName string `json:"priorityClassName,omitempty" protobuf:"bytes,24,opt,name=priorityClassName"`
+	PriorityClassName *string `json:"priorityClassName,omitempty" protobuf:"bytes,24,opt,name=priorityClassName"`
```

Comment:
Where is this protobuf tag coming from? Can this be removed?

Reply:
oh nice catch! thank you!!!

Reply:
Done!